components/layout_2020/flexbox/layout.rs

```diff
@@ -1545,15 +1548,20 @@ impl InitialFlexLineLayout<'_> {
                 let mut fragment_info = item.box_.base_fragment_info();
                 fragment_info.flags.insert(FragmentFlags::IS_FLEX_ITEM);
 
+                let container_writing_mode = flex_context.containing_block.style.writing_mode;
```

Comment:
We use the name `containing_block_writing_mode` in few other places. Does that have the same meaning as `container_writing_mode`?

Reply:
In this case I'm just being a bit more explicit since it is the writing mode of the flex container that we are using. "Containing block" is used liberally in other places, but it isn't exactly accurate -- though this is larger issue we need to fix. For instance, for absolutely positioned elements, they may have a different containing block further up the tree.